src/communication/global_policy.hpp

```diff
@@ -1,21 +1,37 @@
 #pragma once
 
-#ifdef NMC_HAVE_MPI
-    #include "communication/mpi_global_policy.hpp"
+#include <string>
+
+namespace nest { namespace mc { namespace communication {
+    enum class global_policy_kind {serial, mpi, dryrun};
```

Comment:
the `global_policy_kind` is used in the miniapp to test if the application is running in dry run mode at run time. If we used `const char*`, we would have to use `strcmp` to make this test.